backend/src/services/secret/secret-service.ts

```diff
@@ -2521,6 +2521,36 @@ export const secretServiceFactory = ({
     });
   };
 
+  const getSecretVersionsV2ByIds = async ({
+    actorId,
+    actor,
+    actorOrgId,
+    actorAuthMethod,
+    secretId,
+    secretVersions,
+    folderId
+  }: TGetSecretVersionsDTO & {
+    secretVersions: string[];
+    folderId: string;
+  }) => {
+    const secretVersionV2 = await secretV2BridgeService
+      .getSecretVersionsByIds({
+        actorId,
+        actor,
+        actorOrgId,
+        actorAuthMethod,
+        secretId,
+        folderId,
+        secretVersionNumbers: secretVersions
+      })
+      .catch((err) => {
+        if ((err as Error).message === "BadRequest: Failed to find secret") {
+          return null;
+        }
```

Comment:
why do we have this catch block in this reusable method? maybe it makes more sense to have this at a higher level rather than here?